apps/docs/src/pages/brand-colors.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Colors
+description: Sample Colors documentation
+layout: ../layouts/MainLayout.astro
+---
+
+import {
+  ColorWrapper,
+  ColorsWrapper,
+  colorsInfo,
+} from '@selleo/core/src/Colors';
+import { H2 } from '@selleo/core/src/Headers';
+import Preview from '../components/Preview.astro';
+import Description from '../components/Description.astro';
+
+<Description>
+  Below you can find our brand colors styled with tailwindcss.
+</Description>
+
+<H2>Brand Colors</H2>
+
+<div class="preview">
+{colorsInfo.brand.primary.sectionOne.map(({ class: classes, colorInfo, size }) => <ColorWrapper colorInfo={colorInfo} size={size} class={classes} />)}
+
+<ColorsWrapper itemsLength={4} class="mb-7 mt-3">
```

Comment:
Current implementation makes so much duplication? How can we improve is:
- store the colors without section names inside the object 
- make the mapping logic in ColorsWrapper with optional optional prop withHero: Boolean (creates this big color panel as in primary)
- otherwise colors wrapper would just take array of items it needs to display

Reply:
Right, really good! Now it'll be easier to manage colors page